test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -496,7 +480,6 @@ Describe "TabCompletion" -Tags CI {
                 $afterTab = 'filesystem::/usr' -f $env:SystemDrive
             }
             $res = TabExpansion2 -inputScript $beforeTab -cursorColumn $beforeTab.Length
-            $res.CompletionMatches.Count | Should BeExactly 1
```

Comment:
Maybe `Should BeGreaterThan 0` - else next line can throw with useless message.

Reply:
Correct, I confused this with the about_help test. This can be BeGreaterThan 0.

Reply:
Fixed.